build.py

```diff
@@ -20,29 +20,35 @@
 OUTPUT_DIR = 'output'   # Zip artifact directory
 
 
-def build_region(region, path, full_framework, framework_branch=None,
-                 region_branch=None, is_prod=False, is_dev=False):
+def build_region(region, path, full_framework, override_framework_branch=None,
+                 override_region_branch=None, is_prod=False, is_dev=False):
     """ Build a GeositeFramework Region site and zip it up for installation """
     workspace = setup_workspace(path)
 
     # Get the repo name and to avoid confusion name it a _config dir
     region_dest = '%s_config' % region.split('/')[1]
     region_name = get_region_name(region)
 
-    # If --dev was specified, favor a branch named 'development' for the
-    # region repo and any plugins, but don't fail if it doesn't exist
-    # This is the convention used by TNC developers to introduce new features
-    if region_branch and region_branch != DEFAULT_REGION_BRANCH:
-        # An override was provided, use it
-        region_branch = region_branch
-    elif is_prod:
-        # Production build, use master
+    print(override_framework_branch, override_region_branch)
```

Comment:
Is this for debugging?  Should remove if so, otherwise add some labels so that we can see which value is which in the build log (this might be useful).

Reply:
Thanks. Left-over from debugging.